.github/workflows/reusable-examples.yml

```diff
@@ -546,3 +546,45 @@ jobs:
       - name: Build
         working-directory: examples/sveltekit
         run: npm run build
+
+  nextjs-14-sensitive-info:
```

Comment:
Instead of adding to the end of the file, can you insert it between the 2 examples as found in the directory tree? Whenever I need to audit these, I compare with my directory listing.